testing/e2e/app/routes/DropdownMenu/spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('DropdownMenu', () => {
+  test('should support deep level submenus', async ({ page }) => {
```

Comment:
This e2e test seems to be flaky. Any suggestions on how to make it pass reliably?

Reply:
Have you tried debugging it using [`--ui` mode](https://playwright.dev/docs/test-ui-mode)?

Full command:
```
pnpm run build --filter=e2e && pnpm --filter=e2e run test --ui
```

After that, you could play with some delays. Playwright has multiples ways of introduces delays in various places ([example](https://playwright.dev/docs/api/class-keyboard#keyboard-press-option-delay)).

Reply:
Ohh, I see, thanks. The UI mode helped me debug the issue and delays were one of the things that fixed the flakiness (https://github.com/iTwin/iTwinUI/pull/1942/files/7a41f31b492eb3ab656ecb1f8b43bb8bdbc2ac32..9e471ef813e96edd4ef841d7fae70f1c18b6e999).